packages/pigeon/platform_tests/test_plugin/android/src/main/kotlin/com/example/test_plugin/TestPlugin.kt

```diff
@@ -213,6 +209,15 @@ class TestPlugin: FlutterPlugin, HostIntegrationCoreApi {
   override fun callFlutterNoop(callback: (Result<Unit>) -> Unit) {
     flutterApi!!.noop() { callback(Result.success(Unit)) }
   }
+  
+  // TODO: (tarrinneal) Update error handling tests to properly recieve and handle errors.
+  // See issue https://github.com/flutter/flutter/issues/118243
+  override fun callFlutterThrowError(callback: (Result<Any?>) -> Unit) {
+    flutterApi!!.throwError() { echo -> callback(Result.failure(Exception("except"))) }
```

Comment:
Same note here as for the Java code: no code and a TODO is safer since staying wrong won't pass.